src/overlays/actors/ovl_Dm_Zl/z_dm_zl.c

```diff
@@ -130,46 +126,50 @@ void DmZl_Destroy(Actor* thisx, PlayState* play) {
 void DmZl_DoNothing(DmZl* this, PlayState* play) {
 }
 
-void DmZl_UpdateCutscene(DmZl* this, PlayState* play) {
+void DmZl_HandleCutscene(DmZl* this, PlayState* play) {
     s32 cueChannel; // reused as animIndex
```

Comment:
```suggestion
    s32 cueChannel;
```
Isn't needed anymore right?